package_manifest.yaml

```diff
@@ -384,14 +384,6 @@ foreman_nonscl_packages:
         - "jenkins_job=foreman-installer-develop-source-release"
         - "jenkins_url=https://ci.theforeman.org"
       build_package_tito_builder: "tito.builder.FetchBuilder"
-    foreman-proxy:
```

Comment:
It surprises me that this was here. Did we have two entries for foreman-proxy?

Reply:
We did